src/message/message.service.ts

```diff
@@ -77,4 +77,20 @@ export class MessageService {
     const message = await this.findOne(messageId); // throws NotFoundException if not found
     await this.messageRepository.remove(message);
   }
+
+  async getContextForChat(chatId: number): Promise<string[]> {
```

Comment:
Why did you add this function? The `findAll` method gives you all the messages for a given `chatId`